src/core/common/logging.cpp

```diff
@@ -45,7 +45,7 @@
 
 #ifndef WINDOWS_LOGGING
 #define otLogDump(aFormat, ...)                                             \
-    _otPlatLog(aInstance, aLogLevel, aLogRegion, aFormat OPENTHREAD_CONFIG_LOG_SUFFIX, ## __VA_ARGS__)
+    _otDynmLog(aInstance, aLogLevel, aLogRegion, aFormat OPENTHREAD_CONFIG_LOG_SUFFIX, ## __VA_ARGS__)
```

Comment:
nit: could just use `_otDynamicLog` - not sure saving a few characters in this case is needed given that it's only used in a small handful of places.